docs/docs/reference/anomaly_detection.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 1
```

Comment:
I believe adding it here, may require moving other docs elements, also I think would add that more towards the end of the docs